### PR TITLE
40 switch from flask to fastapi

### PR DESCRIPTION
**Pull request by gcarvellas**
Summary of changes:

- we no longer use flask. We now use fastapi
- all synchronous DB calls are replaced with async. There should be no reason why we need a synchronous DB call
- the APIdocs are slightly different but they're mostly the same. Go to `http://localhost:3001/docs` to view
- swagger files are not necessary so they're all removed

### CODE REVIEW

File: database/base_db.py
Comment (gcarvellas) [inline database/base_db.py:31]: This will be done after this gets merged https://github.com/castlepointanime/backend/pull/41

Once that gets merged I'll make an issue to fix the auto approver assigning system

**Review by Yun-L — APPROVED** [review]
tested briefly, seems to run fine

lgtm

**Pull request merged**
